### ISSUE TITLE
Update instructions to set Valid OAuth Redirect URLs

### ISSUE THREAD

**Issue by Dom-Mc** [learn-co-curriculum/devise_lab#19]
Facebook has changed it's setup process for OAuth redirects and can be somewhat tough to find.  The lab instructions should probably be updated to reflect the change(s). I would do it but lately I've been getting error messages back when attempting to make lab/readme edits.

Here's a helpful link:
https://github.com/tschellenbach/Django-facebook/issues/376

**Comment by mendelB** [comment]
Thanks so much for raising this issue @Dom-Mc this has been updated and clarified here #23. Definitely let us know if you have any additions you'd add.

**Comment by Dom-Mc** [comment]
I also got tripped up by not fully understanding how env variables work. I often open 2 tabs in the terminal, going back and forth from the server to the console.  So it took a few attempts to realize why setting the variables in one session wouldn't be picked up by the other.  I noticed a few others were running into the same or similar issues. FYI the [article](https://launchschool.com/blog/managing-environment-configuration-variables-in-rails) you provided yesterday was extremely helpful! Could we add that as a resource? And [here's](http://blog.honeybadger.io/ruby-guide-environment-variables/) another good one I stumbled upon.

I did a tutorial a while back that used examples like `export FACEBOOK_KEY="your_facebook_key"` which was helpful and one person I spoke with wasn't putting quotes around their values.  Not sure if it's necessary but using an example like this could help.
